shared/js/ui/pages/mixins/parse-query-string.es6.js

```diff
@@ -0,0 +1,25 @@
+module.exports = {
```

Comment:
Why is this necessary?

Reply:
It's a mixin, so it's meant to be shareable from multiple places via require()